### ISSUE TITLE
Common: add Snowplow Ecosystem flowchart

### ISSUE THREAD

**Issue by chuwy** [snowplow-incubator/engineering-resources#6]
To show dependency between core libraries
![Snowplow Ecosystem](https://user-images.githubusercontent.com/681171/65569878-adc2fd80-df91-11e9-88a2-efbba1aded9b.png)

**Comment by benjben** [comment]
There is also a transitional dependency on Scala Analytics SDK for Scala Common Enrich, because of `snowplow-badrows` (because the loaders bad rows need `Event`)

**Comment by chuwy** [comment]
Yeah, there's a lot of things missing (all enrich libs: weather, forex, referer parser, Iglu Server etc). This one is just a quick draft. There's a source file in [dependency-flowchart](https://github.com/snowplow-incubator/engineering-resources/tree/dependency-flowchart) branch. I used draw.io - feel free to extend.

**Comment by benjben** [comment]
@chuwy I tried to automatically generate the graph using webgraphviz.com.
You can see the details [here](https://github.com/snowplow-incubator/engineering-resources/blob/dependency-flowchart/img/dependencies.md) and the result [here](https://htmlpreview.github.io/?https://github.com/snowplow-incubator/engineering-resources/blob/dependency-flowchart/img/dependencies.html).

The big advantage is that there is no need to manually bother with the layout, but it doesn't look as good.
